angular.json

```diff
@@ -1286,15 +1287,18 @@
         "build": {
           "builder": "@nrwl/node:package",
           "outputs": [
-            "{options.outputPath}"
+            "dist/schematics"
```

Comment:
I think you should be able to revert this line

Reply:
done